nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -216,7 +216,8 @@ contract PolicyManager is Upgradeable {
         uint256 reward = node.reward;
         require(reward != 0);
         node.reward = 0;
-        _recipient.transfer(reward);
+        (bool success, ) = _recipient.call.value(reward)("");
```

Comment:
It's, of course, terrible if we have to write it this way. But testnet is on Istanbul now.
Is it indeed a problem? And new solc doesn't compile it properly? If so, I'd vote for a merge.
If it 's to be merged, can we add a comment like `// Transfer reward to _recipient`?

Reply:
I don't think that is terrible, looks a little bit ugly, but I don't think "transfer" was the best protection from reentrancy. Even without Istanbul it removes restriction on complexity of fallback function in recepient contract

Reply:
✔️

Reply:
It will increase size definitely, but size is problem for StakingEscrow and a little bit for Adjudicator, so it'll ok for this 4 places

Reply:
✔️